.github/workflows/terratest.yml

```diff
@@ -1,5 +1,8 @@
 name: terratest
-permissions: read-all
+
+permissions:
+  contents: write
+  pull-requests: write
```

Comment:
Why does this terratest job need write access for pull-requests and contents?

Reply:
Discussed with Jai.

Reply:
In case anyone is reading:

* `pull-requests: write` - so Semantic Release can update PRs to include info on what release they're in.
* `contents: write` - so Semantic Release can push tags